config/initializers/setup_mail.rb

```diff
@@ -0,0 +1,10 @@
+ActionMailer::Base.delivery_method = :smtp
+ActionMailer::Base.smtp_settings = {
```

Comment:
This data should probably not be in source control, I'd suggest using the SendGrid Heroku addon or using environment variables for this for security.